src/models/topic-extraction/TopicExtractionGetListOfJobsOptions.ts

```diff
@@ -0,0 +1,5 @@
+/** Topic extraction get list of jobs options */
+export interface TopicExtractionGetListOfJobsOptions {
```

Comment:
This seems unnecessary and like we'll have unnecessary duplication or every api. Can we at least just call it GetListOfJobsOptions and put it in a place where every api using the new client can use it?